app/vmui/packages/vmui/src/utils/uplot/helpers.ts

```diff
@@ -29,10 +29,7 @@ export const defaultOptions = {
 };
 
 export const formatTicks = (u: uPlot, ticks: number[], unit = ""): string[] => {
-  return ticks.map(v => {
-    const n = Math.abs(v);
-    return `${n > 1e-3 && n < 1e4 ? v.toString() : v.toExponential(1)} ${unit}`;
-  });
+  return ticks.map(v => `${v.toLocaleString()} ${unit}`);
```

Comment:
This may result in partially visible or invisible values at Y axis for numbers with big number of decimal digits because of limited space to display such numbers. The proper solution would be to adjust the space to display these numbers depending on the longest displayed number.

Reply:
It looks like this is no longer the case - big numbers are correctly displayed in full on Y axis now.